internal/bgp/frr/frr.go

```diff
@@ -252,17 +252,22 @@ func (sm *sessionManager) createConfig() (*frrConfig, error) {
 		   'neighbor.Advertisements' list. */
 		for _, adv := range s.advertised {
 			var version string
+			community := make([]string, 0)
 			if adv.Prefix.IP.To4() != nil {
 				version = "ipv4"
 			} else if adv.Prefix.IP.To16() != nil {
 				version = "ipv6"
 			}
-
+			for _, c := range adv.Communities {
+				a := c >> 16
```

Comment:
maybe use better names than a and b? or add an explanation what's done here to create the community?

Reply:
sure